### PR TITLE
OETH Morpho Aave v2 Fork tests

### PR DESCRIPTION
**Pull request by shahthepro**
If you made a contract change, make sure to complete the checklist below before merging it in master.

Refer to our [documentation](https://github.com/OriginProtocol/security) for more details about contract security best practices.

Contract change checklist:
  - [ ] Code reviewed by 2 reviewers. 
  - [ ] Copy & paste code review [security checklist](https://github.com/OriginProtocol/security/blob/master/templates/Contract-Code-Review-Example.md) below this checklist.
  - [ ] Unit tests pass
  - [ ] Slither tests pass with no warning
  - [ ] Echidna tests pass if PR includes changes to OUSD contract (not automated, run manually on local)

### CODE REVIEW

File: contracts/test/_fixture.js [hunk 1]
```diff
@@ -815,6 +856,34 @@ async function morphoAaveFixture() {
...
+    const fixture = await oethDefaultFixture();
+
+    if (isFork) {
+      const sGuardian = await ethers.provider.getSigner(
+        addresses.mainnet.Guardian
+      );
+
+      await fixture.oethVault
+        .connect(sGuardian)
+        .setAssetDefaultStrategy(
```
Comment (sparrowDom) [inline contracts/test/_fixture.js:873]: note: if Transfer Governance Pr https://github.com/OriginProtocol/origin-dollar/pull/1541 gets merged in first `sGuardian` will need to be changed to `sGovernor` - old mainnet timelock contract.

**Review by sparrowDom — APPROVED** [review]
LGTM great work

**Comment by sparrowDom** [comment]
The only unit test failing is an unrelated one. Merging this to @DanielVF 's branch

File: contracts/test/_fixture.js [hunk 2]
```diff
@@ -951,13 +1020,24 @@ async function impersonateAccount(address) {
...
+async function _hardhatSetBalance(address, amount = "10000") {
+  await hre.network.provider.request({
+    method: "hardhat_setBalance",
+    params: [
+      address,
+      utils
+        .parseEther(amount)
+        .toHexString()
+        .replace(/^0x0+/, "0x")
+        .replace(/0$/, "1"),
```
[the remark this answers is not in the document]
Comment (shahthepro) [reply contracts/test/_fixture.js:1032]: Yeah it does with both trailing and leading zeroes

**Pull request merged**